src/service/action-impl.js

```diff
@@ -871,7 +872,9 @@ export class ActionService {
     }
 
     if (Object.keys(detail).length > 0) {
-      event.detail = detail;
+      try {
```

Comment:
Can you add a comment here?

Reply:
Sure, added `catch {} // event.detail is readonly`